apps/studio/components/interfaces/BranchManagement/BranchStatusBadge.tsx

```diff
@@ -36,7 +53,7 @@ const BranchStatusBadge = ({ status }: BranchStatusBadgeProps) => {
 
   return (
     <Badge color={isUnhealthy ? 'red' : 'slate'} className="flex items-center gap-1.5">
-      {isUnhealthy && <WarningIcon className="w-3 h-3" />}
+      {isUnhealthy && <WarningIcon className="w-3.5 h-3.5 rounded-full bg-red-1000" />}
```

Comment:
`bg-destructive` please

Reply:
That's just for the background on the icon, <Badge> can only take red, so if I use `bg-destructive` it looks yucky:
<img width="435" alt="Screenshot 2024-03-06 at 21 57 54" src="https://github.com/supabase/supabase/assets/10985857/0af6a1a8-09bb-41a4-b84c-c5129b323fec">

Reply:
ah sorry, lemme check.

Reply:
we don't ship anything without using theme colors now, so still blocked

Reply:
I'll jump in PR, will adjust the Badge component as well

Reply:
update - sent @alaister the codes